build.gradle

```diff
@@ -3,39 +3,67 @@
 buildscript {
 
     ext.kotlin_version = '1.4.10'
```

Comment:
maybe you could use this chance to increment kotlin version to 1.4.31 and coroutine version to 1.4.3?